### PR TITLE
Implement `node:git`

### PR DESCRIPTION
**Pull request by arthurvr**
Let's hope the second time will be better ;) There we go. `node:git` this time. Ref https://github.com/yeoman/generator-node/issues/66.

### CODE REVIEW

File: test/git.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+var helpers = require('yeoman-generator').test;
+
+describe('node:git', function () {
+  before(function (done) {
+    helpers.run(path.join(__dirname, '../generators/git'))
+      .on('end', done);
+  });
+
+  it('creates .gitignore', function () {
+    assert.file([
```
Comment (SBoudrias) [inline test/git.js]: `assert.file('.gitignore');`

Comment (arthurvr) [inline test/git.js]: Sure thing. Not done that way in other generators, though. Updated it.

Comment (SBoudrias) [inline test/git.js]: I'm not always as involve in generator authorship :yum:

**Comment by SBoudrias** [comment]
Thanks!

**Comment by arthurvr** [comment]
My pleasure. Can you tick git on the list?

**Pull request merged**